Snowflake.Data.Tests/Util/SnowflakeDbExceptionAssert.cs

```diff
@@ -41,7 +41,7 @@ public static void HasHttpErrorCodeInExceptionChain(Exception exception, HttpSta
                         case SnowflakeDbException se:
                             return se.ErrorCode == (int)expected;
                         case HttpRequestException he:
-#if NETFRAMEWORK
+#if NETFRAMEWORK || NETCOREAPP
```

Comment:
doesn't it cover all the cases? If so the `#if` is not needed

Reply:
The `if` directive is now removed